webservice/soap/classes/class.ilSoapUtils.php

```diff
@@ -537,31 +537,6 @@ private function linkNode(array $node, ilCopyWizardOptions $cp_options) : int
         return $new_ref_id;
     }
 
-    /**
-     * @todo get rid of the soap wrapper
-     */
-    public function handleECSTasks(string $sid, int $a_server_id)
```

Comment:
It's not just this method that needs to be removed. In addition, the WSDL generation and other places must also be adjusted. I will create a PR for this.